packages/frontend/components/Forms/FormFanart.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react'
+import TextInput from './TextInput'
+import CheckConfirm from './CheckConfirm'
+import PropTypes from 'prop-types'
+import styles from './Form.module.scss'
+
+interface dataType {
+  creator: string
+  image_url: string
+}
+
+interface errorType {
+  creator?: string
+  image_url?: string
+  confirmation?: string
+}
+
+export default function FormFanart({ hidden }) {
+  const [errors, setErrors] = useState<errorType>({})
+
+  function onSubmit(evt: React.FormEvent<HTMLFormElement>) {
+    evt.preventDefault()
+
+    const formData = new FormData(evt.currentTarget)
+
+    const confirmation = !!formData.get('confirmation')
+    formData.delete('confirmation')
+
+    const data = {} as dataType
+    formData.forEach(function (value, key) {
+      data[key] = value
+    })
```

Comment:
I feel like this part is duplicate in all of the forms and could be extracted to a helper function, though I'm not sure if that makes sense. Let me know your opinion on this since you implemented it.

Reply:
Probably a good idea for the form data to object part (line 29-32), I think I'll make it. for the other, I think it is good as it is

Reply:
I think should be done now, let me now if you have another suggestion for this